__tests__/setup/web/index.html

```diff
@@ -99,9 +99,7 @@
             payload: {
               activity: {
                 from: { id: 'bot' },
-                id: Math.random()
-                  .toString(36)
-                  .substr(2, 10),
+                id: Math.random().toString(36).substr(2, 10),
```

Comment:
I'm not sure why this change or the one on [L6](https://github.com/microsoft/BotFramework-WebChat/pull/3459/files#diff-d80acb31566c0894d493dc1fa0587ed6R6) slipped by me, but I can back them out if they're unwanted.

I ran `npm run eslint` from the root but it errored out on a shipped Web Chat package untouched by this PR...

Reply:
both are linting fixes, so completely fine to ignore. linter should also run on commit.